CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ Main (unreleased)
   `AGENT_MODE=flow`. Support for `EXPERIMENTAL_ENABLE_FLOW` will be removed in
   v0.32. (@rfratto)
 
+- The `ebpf_exporter` integration has been removed due to issues with static
+  linking. It may be brought back once these are resolved. (@tpaschalis)
```

Comment:
This is a breaking change, can we list it as one and mention this in the upgrade guide with a slightly more detailed rationale?

Reply:
Also, when making this a breaking change, the notice above this section should be changed to breaking changes instead of deprecations.

Reply:
You mean putting it in a new section, or renaming this one?

Reply:
New section: https://github.com/grafana/agent/blob/main/docs/developer/contributing.md#updating-the-changelog 

```markdown
> **BREAKING CHANGES**: This release has breaking changes. Please read entries carefully and consult the [upgrade guide](https://grafana.com/docs/agent/latest/upgrade-guide/) for specific instructions.

## Breaking changes

- The `ebpf_exporter` integration has been removed due to issues with static
   linking. It may be brought back once these are resolved. (@tpaschalis)

## Deprecations

...
```